tests/unit/amo/reducers/test_reviews.js

```diff
@@ -1219,4 +1221,21 @@ describe(__filename, () => {
       expect(reviewsAreLoading({ reviews: state }, slug)).toBe(false);
     });
   });
+
+  describe('LOCATION_CHANGE', () => {
+    it('resets the view state', () => {
+      let state;
+
+      state = reviewsReducer(
+        undefined,
+        showEditReviewForm({
+          reviewId: 1,
+        }),
+      );
+
+      state = reviewsReducer(undefined, { type: LOCATION_CHANGE });
```

Comment:
typo:

```diff
diff --git a/tests/unit/amo/reducers/test_reviews.js b/tests/unit/amo/reducers/test_reviews.js
index a6636968d..8ee9ae095 100644
--- a/tests/unit/amo/reducers/test_reviews.js
+++ b/tests/unit/amo/reducers/test_reviews.js
@@ -1233,7 +1233,7 @@ describe(__filename, () => {
         }),
       );
 
-      state = reviewsReducer(undefined, { type: LOCATION_CHANGE });
+      state = reviewsReducer(state, { type: LOCATION_CHANGE });
 
       expect(state.view).toEqual({});
     });
```

How did this test ever fail for you? When I asked if you watch your tests fail you said yes but now I'm giving you the side eye 🍌

Reply:
Moving quickly is good! We use code review to catch things like this (which doesn't always work, heh) -- I think it's great to keep moving fast. We also have a great QA team 😄